qxf2/daily-messages
# Added comments endpoint to send comment reviewer message

This PR contains following changes:

1. I have added comments endpoint in the main.py
2. There is comments_reviewer.py file this file will contain every week's i.e. every Thursday's comment reviewer names. This is not the final file. I will update the file before merging the PR in the master.
3. Also I have added two tests in the in the test_main.py file to test this endpoint.

How to test this change:

1. Start the server using command-uvicorn main:app
2. Once the server is up try to access http://localhost:8000/comment-reviewers
3. In case today is Thursday then comment reviewer message will be displayed.
4. Else you will be shown message as Neither today is Thursday or message for today is not available (You can modify today's date  in the comment_reviewer.py file and check if you get correct message
5.  You can run the tests as well python -m pytest tests/test_main.py -sv for this particular change.
